riscv/src/lib.rs

```diff
@@ -189,7 +189,7 @@ pub fn compile_rust_crate_to_riscv_asm(input_dir: &str) -> BTreeMap<String, Stri
     let temp_dir = Temp::new_dir().unwrap();
 
     let cargo_status = Command::new("cargo")
-        .env("RUSTFLAGS", "--emit=asm")
+        .env("RUSTFLAGS", "--emit=asm -g")
```

Comment:
always?

Reply:
Yes, should not have an impact on performance, should it?

Reply:
I guess more instructions?

Reply:
I hope it only adds directives and not instructions.